pkg/controller/kogitobuild/build/trigger.go

```diff
@@ -47,7 +48,7 @@ func StartNewBuild(buildConfig *v1.BuildConfig, client *client.Client) error {
 
 // cancelRunningBuilds cancels any running builds for the given BuildConfig
 func cancelRunningBuilds(buildConfig *v1.BuildConfig, client *client.Client) error {
-	builds, err := client.BuildCli.Builds(buildConfig.Namespace).List(
+	builds, err := client.BuildCli.Builds(buildConfig.Namespace).List(context.TODO(),
```

Comment:
This parameter can be inside the `openshift/build.go` instead. We will always use `context.TODO()`. That's the same behavior of others client wrappers we have as well. Once we change this behavior to allow a change in the context input parameter, we might revaluate. :P